apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -505,22 +513,44 @@ def test(element_html: str, data: pl.ElementTestData) -> None:
         data["partial_scores"][name] = {
             "score": 1,
             "weight": weight,
-            "feedback": f"The correct answer used for grading was {correct_answer_converted}",
         }
+        if show_correct:
+            data["partial_scores"][name]["feedback"] = (
+                f"The correct answer used for grading was {correct_answer_converted}"
+            )
     elif result == "incorrect":
-        data["partial_scores"][name] = {"score": 0, "weight": weight}
+        data["partial_scores"][name] = {
+            "score": 0,
+            "weight": weight,
+        }
         # Get method of comparison, with relabs as default
         comparison = pl.get_enum_attrib(
             element, "comparison", ComparisonType, COMPARISON_DEFAULT
         )
+        feedback = None
+
+        def formatted_answer(answer: float) -> str:
+            return str(
+                pl.string_fraction_to_number(
+                    str(answer),
+                    allow_fractions=allow_fractions,
+                    allow_complex=allow_complex,
+                )
+            )
 
         if comparison is ComparisonType.RELABS:
             rtol = pl.get_float_attrib(element, "rtol", RTOL_DEFAULT)
             atol = pl.get_float_attrib(element, "atol", ATOL_DEFAULT)
             # Get max error according to numpy.allclose()
             eps = np.absolute(correct_answer_converted) * rtol + atol
-            eps += random.uniform(1, 10)
-            answer = correct_answer + eps * random.choice([-1, 1])
+            # eps += random.uniform(1, 10)
+            answer = correct_answer + eps * random.choice([-1.1, 1.1])
```

Comment:
This `eps +=` may trigger some cases where a student answer is extremely out of bounds, and the instructor wants to provide more detailed feedback.